content/docs/hooks-state.md

```diff
@@ -174,7 +174,7 @@ In a function, we can use `count` directly:
 In a class, we need to call `this.setState()` to update the `count` state:
 
 ```js{1}
-  <button onClick={() => this.setState({ count: this.state.count + 1 })}>
+  <button onClick={() => this.setState(state => ({ count: state.count + 1 }))}>
```

Comment:
If you're changing this, you probably want to change the function version below on line 185. But I think we might purposely not use the function versions here for simplicity.